README.md

```diff
@@ -124,6 +124,101 @@ $ sudo make install
 
 ### Deploying the Daemons
 
+#### The Bluzelle Configuration File
+
+The Bluzelle daemon is configured by setting the properties of a JSON
+configuration file provided by the user. This file is usually called
+*bluzelle.json* and resides in the current working directory. To specify a
+different configuration file the daemon can be executed with the -c command
+line argument:
+
+    $ swarm -c peer0.json
+
+The configuration file is a JSON format file, as seen in the following example:
+
+    {
+        "bootstrap_file": "./peers.json",
+        "ethereum": "0xddbd<...>121a",
+        "ethereum_io_api_token": "53IW57FSZSZS3QXJUEBYT8F4YZ9IZFXBPQ",
+        "http_port": 9082,
+        "listener_address": "127.0.0.1",
+        "listener_port": 49152,
+        "log_to_stdout": true,
+        "uuid": "d6707510-8ac6-43c1-b9a5-160cf54c99f5",
+        "max_storage" : "2GB",
+        "logfile_dir" : "logs/",
+        "logfile_rotation_size" : "64K"
+        "logfile_max_size" : "640K",
+        "debug_logging" : false
+    }
+
+where the properties are:
+
+- "bootstrap_file" - the path to a file containing the list of peers in the swarm that this node will be participating in. See below.
+- "debug_logging" - set this value to true to include debug level log messages in the logs
+- "ethereum" - is your Etherium block chain address, used to pay for transactions.
+- "ethereum_io_api_token" - this is used to identify the SwarmDB daemon to Etherscan Developer API (see https://etherscan.io/apis). Use the given value for now, this  property may be moved out the config file in the future.
+- "http_port" - the socket address where SwarmDB will listen for HTTP client requests.
```

Comment:
http_port is only in the peers file

Reply:
OK, does that mean that the http_port is configured from the peers file?

I'll remove it from the config instructions.